kernel/src/common/volatile_slice.rs

```diff
@@ -0,0 +1,52 @@
+use core::ptr;
```

Comment:
What problem does this solve? Did you encounter an issue with the USB descriptors in memory or something?